src/services/item/plugins/embeddedLink/index.ts

```diff
@@ -26,6 +29,23 @@ const plugin: FastifyPluginAsync<GraaspEmbeddedLinkItemOptions> = async (fastify
   // "install" custom schema for validating embedded link items creation
   extendCreateSchema(createSchema);
 
+  // TODO: should we protect this route ?
```

Comment:
I think it should be accessible only to authenticated members. As it will be used only for creating content. So members will need to be authenticated to create.